dmscripts/helpers/email_helpers.py

```diff
@@ -1,5 +1,15 @@
 from mandrill import Mandrill
 
+from dmutils.email import DMNotifyClient
+
+
+def scripts_notify_client(notify_api_key, logger):
```

Comment:
This is becoming a bit bikesheddy, but why not just use `partial`?

Reply:
"a bit"

Reply:
Partial'd.